pkg/ddc/jindofsx/transform.go

```diff
@@ -265,6 +265,7 @@ func (e *JindoFSxEngine) transformMaster(runtime *datav1alpha1.JindoRuntime, met
 			}
 		}
 		// TODO support cos storage
+		mountType := "oss"
```

Comment:
Could you check jindo image version to determine if it supports transformToken?

Reply:
It is a good question, this feature only support 4.6.7

Reply:
I think we have to make it be compatible with previous version before 4.6.7 is not widely used in production.